recipe/meta.yaml

```diff
@@ -6,36 +6,52 @@ package:
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name.replace('-', '_') }}-{{ version }}.tar.gz
-  sha256: 97e63b5603e085c6a56a12a95740010e75d9134e0aab767e0978b53fd8824f13
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name.replace('-', '_') }}-{{ version }}.tar.gz
+    sha256: 97e63b5603e085c6a56a12a95740010e75d9134e0aab767e0978b53fd8824f13
+  # file needed for tests, not included in the pypi tarball
```

Comment:
I would skip `test_version_number.py` instead of downloading the changelog.

Reply:
ok (skipped `test_version_number_match_with_changelog` because there is another test in the file that passes)

Reply:
That other test is useless for us. It's a test for maintainers, not us. Anyway, it's not a big deal.